src/nls/root/strings.js

```diff
@@ -744,5 +744,8 @@ define({
     "DESCRIPTION_USE_THEME_SCROLLBARS"               : "True to allow custom scroll bars",
     "DESCRIPTION_LINTING_COLLAPSED"                  : "True to collapse linting panel",
     "DESCRIPTION_FONT_FAMILY"                        : "Change font family",
-    "DESCRIPTION_FONT_SIZE"                          : "Change font size; e.g, 13px"
+    "DESCRIPTION_FONT_SIZE"                          : "Change font size; e.g, 13px",
+    "DESCRIPTION_OPEN_PREFS_IN_SPLIT_VIEW"           : "False to disable opening preferences file in split view",
+    "DEFAULT_PREFERENCES_JSON_HEADER_COMMENT"        : "/*\n * This is a read-only reference file for what preferences you can\n * override in your own file. Modify your own preferences on the\n * right and if you have any questions, please refer to\n * https://github.com/adobe/brackets/wiki/How-to-Use-Brackets#preferences\n * for more information on how to use preferences inside {APP_NAME}.\n */",
```

Comment:
A bit of refactoring in the lines of: "This is a read-only file with the preferences supported by brackets. Use this as a reference to modify your own preferences file(brackets.json) opened in the right-hand pane. Refer to https://github.com/adobe/brackets/wiki/How-to-Use-Brackets#preferences  for more information on how to use preferences inside Brackets."